asn1acnconstants.h

```diff
@@ -31,6 +31,7 @@ namespace Constants {
 // Shared constants
 
 const char CONTEXT_MENU[] = "Asn1Acn.ContextMenu";
+const char WIZARD_CATEGORY[] = "O.Asn1Acn";
```

Comment:
Indeed it is. I spend couple of minutes to figure it out, but failed. Wizard would work if category were just "Asn1Acn", however all the other category names are prefixed with upper-case letter, so I have decided to follow the convention. This specific entry contained in  asn1acnconstants.h is not used in any place in code, but I've decided to put it here to let know that some wizard category is defined for Asn1Acn, so it would be easier for someone to look for its definiton.